group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorMetrics.java

```diff
@@ -70,11 +70,11 @@ public class GroupCoordinatorMetrics extends CoordinatorMetrics implements AutoC
     public static final String OFFSET_COMMITS_SENSOR_NAME = "OffsetCommits";
     public static final String OFFSET_EXPIRED_SENSOR_NAME = "OffsetExpired";
     public static final String OFFSET_DELETIONS_SENSOR_NAME = "OffsetDeletions";
-    public static final String GENERIC_GROUP_COMPLETED_REBALANCES_SENSOR_NAME = "CompletedRebalances";
-    public static final String GENERIC_GROUP_REBALANCES_SENSOR_NAME = "GenericGroupRebalances";
+    public static final String CLASSIC_GROUP_COMPLETED_REBALANCES_SENSOR_NAME = "CompletedRebalances";
+    public static final String CLASSIC_GROUP_REBALANCES_SENSOR_NAME = "GenericGroupRebalances";
```

Comment:
I should have added my previous comment over here. Let's continue conversation at this place.

I understand that we have only changed the variable name of the sensor here. My question is, should we change "GenericGroupRebalances" to "ClassicGroupRebalances" here if this has just been introduced in 3.7?

Reply:
Indeed, we should rename this one. Missed it.

Reply:
Fixed.